src/main/java/com/patchworkmc/access/AccessTransformation.java

```diff
@@ -6,7 +6,6 @@
 	public static final AccessTransformation NONE = new AccessTransformation(0, 0);
 	public static final AccessTransformation DEFINALIZE = new AccessTransformation(Opcodes.ACC_FINAL, 0);
 	public static final AccessTransformation MAKE_PUBLIC = new AccessTransformation(Opcodes.ACC_PRIVATE | Opcodes.ACC_PROTECTED, Opcodes.ACC_PUBLIC);
-	public static final AccessTransformation DEFINALIZE_MAKE_PUBLIC = new AccessTransformation(Opcodes.ACC_FINAL | Opcodes.ACC_PRIVATE | Opcodes.ACC_PROTECTED, Opcodes.ACC_PUBLIC);
```

Comment:
why remove this? it's API, no need to prog:arr:d it

Reply:
Eventually I'd like to completely remove/rework the builtin access transformer system, I'm just deleting unused stuff as I go. It seems like we'll only need to make classes public & definalize fields in the future, so this whole system will probably be dropped anyways in favor of something much simpler.

Reply:
Alright, works for me.